### ISSUE TITLE
getProfile | Only getting user id and name

### ISSUE THREAD

**Issue by firasalkafri** [sromku/android-simple-facebook#389]
I am getting only userId and name in onComplete of onProfileListener rest entities like gender,bio, birthday, cover are null.

It seems to be a bug cause I also tried the sample code and got the exact same problem.

- Using Graph v2.5
- Asked for the right permissions
- Specified profile properties in my request

**Comment by master1234** [comment]
Yes I also start to have this issue. It is very weird. No more email I get just first name last name and id.

**Comment by Savrov** [comment]
just use response.getEmail().toString()...and so on
